common/text-level-modifiers/text-level-modifiers-en.html

```diff
@@ -0,0 +1,84 @@
+---
+{
+	"title": "Text-Level Modifiers",
+	"language":"en",
+	"altLangPage":"text-level-modifiers-fr.html",
+	"breadcrumbs":
+		[
+			{
+				"title": "GCWeb home",
+				"link": "https://wet-boew.github.io/themes-dist/GCWeb/index-en.html"
+			}
+		],
+	"secondlevel": false,
+	"dateModified": "2023-07-20",
+	"share": "true"
+}
+---
+
+<h3>CSS Class</h3>
```

Comment:
The page is generated with an h1, so the next heading level need to be an h2. The heading sequence must be sequential.